elbs/virginia/snippets-virginia.tfvars

```diff
@@ -0,0 +1,6 @@
+snippets_elb_name = "snippets"
+snippets_subnets = "subnet-43125f6e,subnet-a699aaef,subnet-b6ceb6ed"
+snippets_instances = "i-007cad57f0c5b3157,i-01fd2b1570e510dfe,i-04c56e5c677568159,i-0981f33bdf95c3c1f,i-0f71cd35d3bb09512"
```

Comment:
sounds, good, option 3 it is then. something along the lines of

```sh
aws autoscaling attach-load-balancers \
      --auto-scaling-group-name $ASG \
      --load-balancer-names $ELB
```

should be fine